src/test/java/com/dsrm/dsrmbackend/admin/AdminReservationControllerTests.java

```diff
@@ -52,14 +53,14 @@ class AdminReservationControllerTests extends  AbstractIntegrationTest{
     @Test
     @Transactional
     void addValidReservation() throws Exception {
-        Map<String,Object> body = new HashMap<>();
-        body.put("room",1);
-        body.put("from","2023-02-21 12:20:00");
-        body.put("to","2023-02-22 13:20:00");
-        body.put("user",2);
+        ReservationRequestDTO reservationRequestDTO  = new ReservationRequestDTO();
+        reservationRequestDTO.setRoom("1");
+        reservationRequestDTO.setFrom(LocalDateTime.parse("2023-07-21T12:20:00"));
```

Comment:
We should use current date plus some time. Because after 21.07.2023 it will stop working